### ISSUE TITLE
 dll load failed 1 is not a valid win32 application

### ISSUE THREAD

**Issue by enderphan94** [corelan/mona#36]
When opening a new issue, please fill out the following sections:

### Expected behavior 
!mona modules should be running
### Actual behavior
Error dll load failed 1 is not a valid win32 application
### Steps to reproduce the problem
1, Donwload mona
2, Copy it to pycommands of ID
3, run `!mona modules`
4, Got error:  dll load failed 1 is not a valid win32 application

### Other useful information (mona version, debugger & debugger version, OS version, etc)
Mona: Newest version
ID: v1.85
OS: Windows 10 (64 bits)

**Comment by corelanc0d3r** [comment]
can you please check/confirm that mona.py is in fact the python code?
did you change the properties of mona.py and marked it "safe" (unblock) ?

what happens if you run !mona without arguments?

**Comment by enderphan94** [comment]
I solved this issue by using windows xp, and it works. It could not work in windows 64 bits?. I don't really know. I will try your suggestion out later and let you know.

**Comment by corelanc0d3r** [comment]
It works fine on Windows 10 x64 too - Immunity is still 32bit - I use it all the time on Win10 x64

**Comment by Nihlander** [comment]
I've just encountered this error as well...

## Environment
Operating System: Windows 7 Pro SP1 (64-bit)
Immunity Debugger: v1.85 (32-bit)
Python: 2.7.16 (v2.7.16:413a49145e, Mar  4 2019, 01:37:19) [MSC v.1500 64 bit (AMD64)] on win32

## Details
Immunity Debugger is running as admin.
Mona.py extracted from download of zip archive of mona repo.
Applied unblock in properties of mona.py (Unsure if this has actually done anything though, button is still there.)

## Screenshot
Any attempt to execute mona from within Immunity Debugger produces this error...

![image](https://user-images.githubusercontent.com/16904792/62289796-9404ae80-b457-11e9-9866-21b9fb8d66f0.png)

**Comment by corelanc0d3r** [comment]
That is correct. Immunity is 32bit and requires 32bit python. Thanks for confirming
